examples/artemis-transaction-consumer/artemis_transaction_consumer.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/artemis;
+import ballerina/io;
+
+public function main() returns error? {
+    // Create the Artemis listener with `autoCommitAcks` set to `false`.
+    // In this case the message acknowledgements will be committed only
```

Comment:
```suggestion
    // In this case, the message acknowledgments will be committed only
```

Reply:
I guess the correct spelling is acknowledgements